test/lib/seedbank/dsl_test.rb

```diff
@@ -74,40 +74,45 @@
   end
 
   describe 'define_seed_task' do
+    subject { Seedbank::DSL.define_seed_task(seed_file, task_name => dependencies) }
+
     let(:task_name) { 'scoped:my_seed' }
-    let(:dependencies) { ['environment'] }
+    let(:dependencies) { ['scoped:another_seed'] }
```

Comment:
I defined some dependencies to check that everything will work fine with them.
But within the gem `define_seed_task` is never called with any dependencies, only with task name.